src/cargo-fmt/main.rs

```diff
@@ -122,8 +122,19 @@ fn handle_command_status(status: Result<i32, io::Error>, opts: &getopts::Options
     }
 }
 
-fn get_version(verbosity: Verbosity) -> Result<i32, io::Error> {
-    run_rustfmt(&BTreeSet::new(), &[String::from("--version")], verbosity)
+fn get_version() -> Result<i32, io::Error> {
+    let mut status = vec![];
+    let mut command = Command::new("rustfmt")
+        .stdout(std::process::Stdio::inherit())
+        .args(&[String::from("--version")])
+        .spawn()?;
```

Comment:
Can you add the error mapping ?

https://github.com/rust-lang/rustfmt/pull/3506/files#diff-0f04fceb10cfdeda3716c7824a8dc2deL361

Reply:
Thank you the review.
I added the error mapping.